packages/cli/src/test-utils/multisigUtils.ts

```diff
@@ -0,0 +1,56 @@
+import { multiSigABI, proxyABI } from '@celo/abis'
+import { StrongAddress } from '@celo/base'
+import { ContractKit } from '@celo/contractkit'
+import { multiSigBytecode, proxyBytecode } from './constants'
+
+export async function createMultisig(
+  kit: ContractKit,
+  owners: StrongAddress[],
+  requiredSignatures: number,
+  requiredInternalSignatures: number
+): Promise<StrongAddress> {
+  const accounts = (await kit.web3.eth.getAccounts()) as StrongAddress[]
+  kit.defaultAccount = accounts[0]
+
+  // Deploy Proxy contract
+  const proxyDeploymentTx = await kit.sendTransaction({
+    data: proxyBytecode,
+  })
+  const { contractAddress: proxyAddress } = await proxyDeploymentTx.waitReceipt()
+  // Deploy MultiSig contract
+  const multisigDeploymentTx = await kit.sendTransaction({
+    data: multiSigBytecode,
+  })
+  const { contractAddress: multiSigAddress } = await multisigDeploymentTx.waitReceipt()
+
+  // Configure and initialize MultiSig
+  const initializerAbi = multiSigABI.find(
+    (abi) => abi.type === 'function' && abi.name === 'initialize'
+  )
+  const proxy = new kit.web3.eth.Contract(proxyABI as any, proxyAddress)
+  const baseFee = await kit.web3.eth.getBlock('latest').then((block: any) => block.baseFeePerGas)
+  const priorityFee = kit.web3.utils.toWei('2', 'gwei')
+  const initMethod = proxy.methods._setAndInitializeImplementation
+  const callData = kit.web3.eth.abi.encodeFunctionCall(initializerAbi as any, [
+    owners as any,
+    requiredSignatures as any,
+    requiredInternalSignatures as any,
+  ])
+  const initTx = initMethod(multiSigAddress, callData)
+  await initTx.send({
+    from: kit.defaultAccount,
```

Comment:
you shouldnt need to the the from.

Reply:
Good point, when I remove it I get this error

```sh
$ yarn workspace @celo/celocli jest propose.test.ts --detectOpenHandles

...

 No "from" address specified in neither the given options, nor the default options.

      38 |   ])
      39 |   const initTx = initMethod(multiSigAddress, callData)
    > 40 |   await initTx.send({
         |                ^
      41 |     gas: await initTx.estimateGas({ from: kit.defaultAccount }),
      42 |     maxPriorityFeePerGas: priorityFee,
      43 |     maxFeePerGas: (BigInt(baseFee) + BigInt(priorityFee)).toString(),
```

Although I specify a default account above 

https://github.com/celo-org/developer-tooling/blob/a20e707e9af56ff03e95e4a8f2fd64b6bae06065/packages/cli/src/test-utils/multisigUtils.ts#L13-L13

Not 100% sure what's wrong, but my hunch is it's acceptable to leave the `from` address. What do you think? Any ideas what might be wrong?

Reply:
a special case maybe. no worries